src/main/java/org/mastodon/mamut/feature/branch/BranchNSubBranchSpotsFeature.java

```diff
@@ -0,0 +1,110 @@
+package org.mastodon.mamut.feature.branch;
+
+import static org.mastodon.feature.FeatureProjectionKey.key;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.mastodon.feature.Dimension;
+import org.mastodon.feature.Feature;
+import org.mastodon.feature.FeatureProjection;
+import org.mastodon.feature.FeatureProjectionKey;
+import org.mastodon.feature.FeatureProjectionSpec;
+import org.mastodon.feature.FeatureProjections;
+import org.mastodon.feature.FeatureSpec;
+import org.mastodon.feature.IntFeatureProjection;
+import org.mastodon.feature.Multiplicity;
+import org.mastodon.mamut.model.branch.BranchSpot;
+import org.mastodon.properties.IntPropertyMap;
+import org.scijava.plugin.Plugin;
+
+/**
+ * Represents the total number of successors of a branch spot in the whole track sub-tree of this branch spot.
+ * <p>
+ * In the following example this number would equal to following branchSpots as
+ * follows:
+ *
+ * <pre>
+ *                         branchSpot0
+ *  	       ┌──────────────┴─────────────────┐
+ *  	       │                                │
+ *  	   branchspot1                      branchSpot2
+ *  	┌──────┴───────┐
+ *  	│              │
+ *  branchspot3 branchSpot4
+ * </pre>
+ *
+ * <ul>
+ * <li>{@code branchSpot0 = 4}</li>
+ * <li>{@code branchSpot1 = 2}</li>
+ * <li>{@code branchSpot2 = 0}</li>
+ * <li>{@code branchSpot3 = 0}</li>
+ * <li>{@code branchSpot4 = 0}</li>
+ * </ul>
+ */
+public class BranchNSubBranchSpotsFeature
```

Comment:
I would like to keep this feature even though there is a strict mathematical relationship between it and the BranchNLeavesFeature.

This mathematical relationship might not be so obvious for the every user of Mastodon and for some the one feature may be easier to understand and for some others the other feature may be easier to understand. Yes, they could be easily converted one to the other by the formula that you write, but I would like to avoid handing over this transformation step to the users.

Regarding the machine learning, I agree. Features that strongly inter-depend from each other may negatively influence the results. However, the Features to be included in the machine learning will need to manually selected anyway.

Reply:
I'm ok with keeping this feature. I like that the name and description of this feature are now so much simpler :+1: